packages/cosmos_ui_components/lib/components/cosmos_app_bar.dart

```diff
@@ -29,22 +31,24 @@ class CosmosAppBar extends StatelessWidget implements PreferredSizeWidget {
               if (actions != null) ...actions!,
             ],
           ),
-          const SizedBox(height: 20),
```

Comment:
you have now completely removed the SizedBox, which is needed if we have the title set. please add it back and just move it under the if statement:
```dart
if (title != null) 
  ...[
    SizedBox(...),
    Padding(child: Text(...),...),
  ],

Reply:
Yup, missed that